server/repository/src/db_diesel/assets/asset_internal_location_row.rs

```diff
@@ -117,7 +117,7 @@ impl<'a> AssetInternalLocationRowRepository<'a> {
 
 impl Upsert for AssetInternalLocationRow {
     fn upsert_sync(&self, con: &StorageConnection) -> Result<(), RepositoryError> {
-        let _change_log_id = AssetInternalLocationRowRepository::new(con).upsert_one(self)?;
```

Comment:
Personally I prefer this version, as it clearly identifies what we're throwing away.